style.css

```diff
@@ -82,12 +84,349 @@ h1 {
+}
+
+/* About me section */
+.aboutMe {
+  width: 375px;
+  height: 1650px;
+  background-color: #1c1a19;
+  display: flex;
+  flex-direction: column;
+  margin-top: 120px;
+  padding: 120px 24px;
+}
+
+.aboutMeTitle {
+  width: 327px;
+  height: 79px;
+  color: #ff6b00;
+  font-family: 'Crete Round', Georgia, Times, 'Times New Roman', serif;
+  font-style: normal;
+  font-weight: 400;
+  font-size: 72px;
+  line-height: 88px;
+  letter-spacing: 0.37px;
+}
+
+.aboutMeDescription {
+  width: 327px;
+  height: 148px;
+  color: #fff;
+  font-family: 'Inter', 'Trebuchet MS', sans-serif;
+  font-style: normal;
+  font-weight: 400;
+  font-size: 16px;
+  line-height: 24px;
+  padding-top: 12px;
+}
+
+.resumeLinkContainer {
+  padding-top: 36px;
+}
+
+.resumeLink {
+  width: 156px;
+  height: 48px;
+  background-color: #ff6b00;
+  border: none;
+  color: #fff;
+  padding: 12px;
+  text-align: center;
+  display: inline-block;
+  font-weight: 400;
+  font-size: 15px;
+}
+
+.developer {
+  width: 352px;
+  display: flex;
+  justify-content: end;
+}
+
+.line2 {
+  padding-top: 6px;
+}
+
+.skills {
+  color: #fff;
+  text-align: center;
+  padding-top: 80px;
+}
+
+.skills ul {
+  font-family: 'InaiMathi', 'Gill Sans', sans-serif;
+  font-style: normal;
+  font-weight: 700;
+  font-size: 32px;
+  line-height: 44px;
+}
+
+.ulList {
+  font-family: 'InaiMathi', 'Gill Sans', sans-serif;
+  font-style: normal;
+  font-weight: 400;
+  font-size: 22px;
+  line-height: 43px;
+  padding-top: 24px;
+  padding-bottom: 24px;
+}
+
+/* contact section */
+.contact {
+  width: 375px;
+  height: 750px;
+  background-color: #fff;
+  display: flex;
+  flex-direction: column;
+  padding-left: 24px;
+}
+
+.newProjects {
+  width: 327px;
+  color: #172b4d;
+  font-family: 'InaiMathi', 'Gill Sans', sans-serif;
+  font-style: normal;
+  font-weight: 700;
+  font-size: 32px;
+  line-height: 44px;
+  text-align: center;
+  padding-top: 50px;
+}
+
+form {
+  padding-top: 50px;
+}
+
+input[type=text] {
+  width: 327px;
+  padding: 12px;
+  margin: 8px 0;
+  display: inline-block;
+  border: none;
+  border-bottom: 1px solid #ccc;
+  box-sizing: border-box;
+}
+
+textarea {
+  background-color: #fbf8f7;
+  border: none;
+  width: 327px;
+  padding: 12px;
+  margin-top: 12px;
+  font-style: italic;
+}
+
+.submitContainer {
+  display: flex;
+  justify-content: center;
+}
+
+input[type=submit] {
+  width: 131px;
+  height: 48px;
+  background-color: #ff6b00;
+  border: none;
+  color: #fff;
+  padding: 12px;
+  font-weight: 400;
+  font-size: 15px;
+  margin-top: 12px;
+}
+
+/* footer section */
+footer {
+  width: 375px;
+  height: 50px;
+  background-color: #fff;
+  display: flex;
+  flex-direction: column;
+}
+
+.socialmedia2 {
+  width: 200px;
+  height: 40px;
+  display: flex;
+  padding-top: 24px;
+  justify-content: space-between;
+  margin-left: auto;
+  margin-right: auto;
+}
+
+.bottomLine {
+  width: 134px;
+  padding-top: 30px;
+  margin-left: auto;
+  margin-right: auto;
```

Comment:
- [ ] kindly consider removing the scroll on the x-axis on the whole page and make sure your page does not scroll horizontally anymore this will give the best user experience
- check on the below image 👇 
![go](https://user-images.githubusercontent.com/60146030/227638299-4b5de2a8-b50b-4ec0-a354-e1f415f8251a.PNG)